test/parallel/test-zlib-sync-no-event.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const zlib = require('zlib');
+
+const zipper = new zlib.Gzip();
```

Comment:
If you use `zlib.gzipSync()` then the object that emits the close event cannot be accessed by the test.

So instead, this is emulating `gzibBufferSync()` and calling `_processChunk()` without a callback so that it can have access to the event if it fires.